solidity/random-beacon/test/RandomBeacon.Pool.test.ts

```diff
@@ -1,18 +1,22 @@
-import { ethers, waffle } from "hardhat"
-import { expect } from "chai"
+/* eslint-disable @typescript-eslint/no-unused-expressions */
 
-import { Signer, Contract } from "ethers"
-import { randomBeaconDeployment } from "./fixtures"
+import { ethers, waffle, helpers } from "hardhat"
+import { expect } from "chai"
+import { ContractTransaction } from "ethers"
```

Comment:
Let's cleanup linter warnings from this file.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2707/commits/c97dba585f021ce3e8ec7fb37e2665a4f7577541 as part of #2707